Assets/VRTK/Scripts/VRTK_MoveInPlace.cs

```diff
@@ -0,0 +1,387 @@
+﻿// Move In Place|Scripts|0145
+namespace VRTK
+{
+    using UnityEngine;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Move In Place allows you to move the play area by calculating the y-movement of your controllers and propelling you the more they are moving.  This simulates walking in game by walking in real life.
```

Comment:
refrain from talking directly about the user as `you`

reword to things like `Move In Place allows the user to move...`  -- `and propelling the user further with more movement`